### PR TITLE
Fix usage of StringUtils and fix ruling test

### PR DESCRIPTION
**Pull request by mstachniuk**

### CODE REVIEW

File: iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/ArmBaseConverter.java [hunk 1]
```diff
@@ -181,7 +180,7 @@ public String filenameAndPosition(TextRange textRange) {
     String position = textRange.start().line() + ":" + textRange.start().lineOffset();
     if (inputFileContext != null) {
       String filename = inputFileContext.inputFile.filename();
-      if (StringUtils.isNotBlank(filename)) {
+      if (filename != null) {
         return filename + ":" + position;
       }
```
Comment (rudy-regazzoni-sonarsource) [inline iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/ArmBaseConverter.java:185]: I would be tempted to also check if the string is blank, like it was done with `StringUtils`.
```suggestion
      if (filename != null && !filename.isBlank()) {
        return filename + ":" + position;
      }
```

**Review by rudy-regazzoni-sonarsource — APPROVED** [review]
LGTM! Please see my small comment.

File: iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/ArmBaseConverter.java (hunk 1 above)
Comment (mstachniuk) [reply iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/ArmBaseConverter.java]: The tests are green, but ok let's do it

**Review by rudy-regazzoni-sonarsource — APPROVED** [review]
LGTM!

**Pull request merged**